src/ledger/backend/contracts/hardhat.d.ts

```diff
@@ -0,0 +1,107 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
Why do we suddenly generate a hardhat file? Do we need/want it?

Reply:
AFAIK this is for better hardhat integration and contains lots of helper types for writing tests. Might come in handy later down the road, but I have no problem with removing it.

Reply:
I'm happy to keep it :)